common/djangoapps/third_party_auth/saml.py

```diff
@@ -41,10 +41,18 @@ def auth_url(self):
         if not self._config.enabled:
             log.error('SAML authentication is not enabled')
             raise Http404
+
         # TODO: remove this check once the fix is merged upstream:
         # https://github.com/omab/python-social-auth/pull/821
```

Comment:
@mattdrayer Nice, thanks for checking. Do you think we should remove that hack and bump the python-social-auth version in a later PR?